### DISCUSSION TITLE
Binaries

### DISCUSSION THREAD

**Discussion by fabiolanza** [JakubMelka/PDF4QT#36] [General]
Hi, would be great to be able to download the binaries already compiled. Are they available anywhere? Thanks.

**Comment by JakubMelka** [comment]
Hello, yes, they are in Releases section for MS Windows:
https://github.com/JakubMelka/PDF4QT/releases

They are in a zip file. Do you need binaries for another system?

**Reply by fabiolanza** [reply]
No, I need it for Windows 11. I see that binaries are not signed though. Are you planning to sign them? Otherwise Windows security keeps barking...

**Reply by JakubMelka** [reply]
Well, could you please give me a hint (or link) how to do that? Where I can get a certificate for signing?

**Reply by JakubMelka** [reply]
@fabiolanza I have created a new release, which have signed binaries (both MSI and EXE files).

**Reply by fabiolanza** [reply]
Thanks, I will download it.

**Comment by JakubMelka** [comment]
The binaries are now signed.
